src/cli.rs

```diff
@@ -0,0 +1,56 @@
+mod account;
+mod balance;
+mod csv;
+mod register;
+
+use crate::error::Error;
+use pargs;
+use std::env;
+
+pub fn run() -> Result<(), Error> {
+    let args: Vec<String> = env::args().collect();
+
+    // define expected args for pargs
+    let command_args: Vec<String> = vec![
+        String::from("account"),
```

Comment:
I like this change going to singular. Clean and consistent with the other commands.